pyccl/neutrinos.py

```diff
@@ -1,33 +1,61 @@
+import numpy as np
 from pyccl import ccllib as lib
+from pyccl.pyutils import check
 
-def Omeganuh2(a,Neff,mnu,TCMB):
+
+def Omeganuh2(a, Neff, mnu, TCMB=2.725):
     """Omeganuh2
     
     Args:
         a (float): Scale factor, normalized to 1 today.
         Neff (float): Number of relativistic neutrino species
         mnu (float): Neutrino mass (in eV)
-        TCMB (float): Temperature of the CMB (K)
+        TCMB (float, optional): Temperature of the CMB (K). Default: 2.725.
     Returns:
         Omeganuh2 (float or array_like)
 
     """
-    status=0
-    return lib.Omeganuh2(a,Neff,mnu,TCMB,None,status)[0]
+    status = 0
+    scalar = True if isinstance(a, float) else False
+    
+    # Convert to array if it's not already an array
+    if not isinstance(a, np.ndarray):
+        a = np.array([a,]).flatten()
+    
+    # Call function
+    OmNuh2, status = lib.Omeganuh2_vec(Neff, mnu, TCMB, a, a.size, status)
+    
+    # Check status and return
+    check(status)
+    if scalar: return OmNuh2[0]
+    return OmNuh2
 
 
-def Omeganuh2_to_Mnu(a,Neff,OmNuh2,TCMB):
+def Omeganuh2_to_Mnu(a, Neff, OmNuh2, TCMB=2.725):
     """Omeganu2h_to_Mnu
 
     Args:
         a (float or array_like): Scale factor(s), normalized to 1 today.
         Neff (float): Number of relativistic neutrino species
         OmNuh2 (float): Neutrino energy density times h^2
```

Comment:
Again I think it would be good to make it very clear how this is different from ccl_omega_x in the doc here.